apps/gp2-frontend/package.json

```
@@ -6,11 +6,10 @@
     "node": "18.x"
   },
   "scripts": {
-    "start": "cross-env TSC_COMPILE_ON_ERROR=true PORT=4000 react-scripts start",
-    "build": "node ./scripts/build-unsupported-browser-page.js && DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "start": "cross-env TSC_COMPILE_ON_ERROR=true PORT=4000 vite start",
```

same as the comment above

```suggestion
    "start": "cross-env TSC_COMPILE_ON_ERROR=true PORT=4000 vite",
```